src/types/Todo.ts

```diff
@@ -1,6 +1,6 @@
 export interface Todo {
-  id: number;
-  userId: number;
-  title: string;
-  completed: boolean;
+  id?: number;
+  userId?: number;
+  title?: string;
+  completed?: boolean;
```

Comment:
why do you make everything optional
```suggestion
  id: number;
  userId: number;
  title: string;
  completed: boolean;
```